src/core/rewrite.ml

```diff
@@ -202,20 +194,21 @@ let break_prod : term -> term * tvar array = fun a ->
     | a         -> (a, Array.of_list (List.rev vs))
   in aux a []
 
-(** [match_pattern (xs,p) t] attempts to match the pattern [p] (containing the
-    “pattern variables” of [xs]) with the term [t]. If successful,  it returns
-    [Some(ts)] where [ts] is an array of terms such that substituting elements
-    of [xs] by the corresponding elements of [ts] in [p] yields a term that is
-    equal to [t] (in terms of [eq]). *)
+(** [match_pattern (xs,p) t] attempts to match the pattern [p] (containing
+   the “pattern variables” of [xs]) with the term [t]. If successful, it
+   returns [Some(ts)] where [ts] is an array of terms such that substituting
+   elements of [xs] by the corresponding elements of [ts] in [p] yields a term
+   that is equal to [t] (in terms of [eq]). *)
```

Comment:
Why do you mess up the comment alignment? You did not even modify anything here!

Reply:
My editor...